addons/im_livechat/static/src/core/common/channel_member_model_patch.js

```diff
@@ -4,7 +4,7 @@ import { patch } from "@web/core/utils/patch";
 /** @type {import("models").ChannelMember} */
 const channelMemberPatch = {
     get name() {
-        if (this.thread.channel_type === "livechat" && this.persona.user_livechat_username) {
+        if (this.thread?.channel_type === "livechat" && this.persona.user_livechat_username) {
```

Comment:
I suppose the `this.thread.anonymous_name` in other branching could potentially be an issue?
I would `?.` also to avoid a future potential traceback.